### PR TITLE
Support Modifying Record w/ Multiple Images

### PR DESCRIPTION
**Pull request by sprice**
Ready for review.

- [x] modify main image
- [x] refactor/cleanup/documentation
- [x] delete existing images
- [x] calculate hashes on FE [name, description, mainImage done]
- [x] calculate fileHashes on FE
- [x] display image thumbnails when viewing record
- [x] click on each thumbnail to swap out viewed image
- [x] styles/css for swapping images

### CODE REVIEW

File: src/components/modals/RecordManageModal.vue [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  },
+  computed: {
+    web3() {
+      return this.$store.state.web3
+    },
+    recordContract() {
+      return this.web3.recordContractInstance()
+    },
+    tokenId() {
+      return this.codexRecord.tokenId
```
Comment (johnhforrest) [inline src/components/modals/RecordManageModal.vue]: Why is this and the next one computed?

File: src/components/modals/RecordManageModal.vue [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    },
+    providerMetadataId() {
+      return this.codexRecord.metadata.id
+    },
+    images() {
+      const images = []
```
Comment (johnhforrest) [inline src/components/modals/RecordManageModal.vue]: This also doesn't look like it needs to be a `computed`. Also suggest renaming it to imageHashes

File: package.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
     "truffle-contract": "^3.0.5",
     "vue": "^2.5.2",
     "vue-router": "^3.0.1",
+    "vue2-dropzone": "^3.2.2",
```
Comment (johnhforrest) [inline package.json]: Not a blocker here, but in general let's try to keep a high standard on importing new dependencies. I've seen it get out of control which leads to bloat and makes the code harder to maintain. Not sure what the correct metrics are we should use to evaluate this, but just something to keep in mind in the future.

File: .env [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# duplicate this file, rename it as .env, and tailor to your local environment
+
+# MIXPANEL_TOKEN=7e7d9ec33e0aa5bf91fc3abbec9c5086
```
Comment (johnhforrest) [inline .env]: Your personal token snuck in. Make sure this file is in the .gitignore

File: src/components/modals/RecordManageModal.vue [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+      const { uuid } = file.upload
+      this.removeAddedImage(uuid)
+    },
+    focusModal() {
+      this.$refs.defaultModalFocus.focus()
+    },
+    clearModal() {
+      Object.assign(this.$data, this.$options.data.apply(this))
+    },
+    updateMetadata() {
```
[the remark this answers is not in the document]
Comment (sprice) [reply src/components/modals/RecordManageModal.vue]: updated.

**Pull request merged**